app/src/main/java/com/web3auth/app/MainActivity.kt

```diff
@@ -135,30 +135,29 @@ class MainActivity : AppCompatActivity(), AdapterView.OnItemClickListener {
             print(ex)
         }
 
-        runOnUiThread {
```

Comment:
This was correct, revert it and the change that caused it.

Reply:
I know that was correct, but I have some changes in the session-manager and in this sdk. So dapps need minimalist changes while upgrading sdk.